docs/index.md

```diff
@@ -20,6 +20,44 @@ for working with [Google Cloud Storage](https://cloud.google.com/storage/)
 Common operations are exposed via the
 [`StorageClient`](obj/api/Google.Storage.V1.StorageClient.yml) class.
 
+Quick start:
+```bat
+PM> Install-Package Google.Storage.V1 -Pre
+```
+
+Sample code:
+```csharp
```

Comment:
We should be able to include this code using a Markdown reference to the snippet. Will get to that when we've done everything else.